src/App.tsx

```diff
@@ -1,34 +1,31 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import './App.scss';
 
-interface State {
+type Props = {
   movies: Movie[];
-}
+};
 
-export class App extends React.Component<{}, State> {
-  state: State = {
-    movies: moviesFromServer,
-  };
+export const App: React.FC<Props> = () => {
+  const [movies, setMovies] = useState<Movie[]>([...moviesFromServer]);
 
-  addMovie = (/* movie: Movie */) => {
-    // put your code here
+  const addMovie = (newMovie: Movie) => {
+    setMovies((prev) => ([
```

Comment:
Не скорочуй назви змінних
```suggestion
    setMovies((prevMovies) => ([
```